backend/src/main/java/com/bakdata/conquery/apiv1/forms/arx/ArxForm.java

```diff
@@ -60,33 +58,48 @@ public class ArxForm extends Form {
 	@DecimalMin(value = "0")
 	private double suppressionLimit = 0.02;
 
-
-	/**
-	 * @see ARXAnonymizer#setMaximumSnapshotSizeDataset(double)
-	 */
-	@Getter
-	@DecimalMax(value = "1")
-	@DecimalMin(value = "0", inclusive = false)
-	private double maximumSnapshotSizeDataset = 0.2;
-
 	/**
-	 * @see ARXAnonymizer#setMaximumSnapshotSizeSnapshot(double)
-	 */
-	@Getter
-	@DecimalMax(value = "1")
-	@DecimalMin(value = "0", inclusive = false)
-	private double maximumSnapshotSizeSnapshot = 0.2;
-
-	/**
-	 * @see ARXAnonymizer#setHistorySize(int)
+	 * The PrivacyModel to use
 	 */
+	@NotNull
 	@Getter
-	@Min(1)
-	private int historySize = 200;
+	private PrivacyModel privacyModel;
 
 	@JsonIgnore
 	private ManagedQuery queryGroup;
 
+	public enum PrivacyModel {
```

Comment:
Macht es denn wirklich Sinn solche Config in ein einzelnes Flag zu quetschen?

Reply:
Ich kann sie in die Config packen denke ich :)